packages/phpstan-rules/src/Rules/NoSuffixValueObjectClassRule.php

```diff
@@ -21,7 +21,7 @@ final class NoSuffixValueObjectClassRule extends AbstractSymplifyRule
     /**
      * @var string
      */
-    public const ERROR_MESSAGE = 'Value Object class name "%s" must be withotu "ValueObject" suffix. The correct class name is "%s".';
+    public const ERROR_MESSAGE = 'Value Object class name "%s" must be without "ValueObject" suffix. The correct class name is "%s".';
```

Comment:
IIRC, there is a rector somewhere which fixes common typos? 

should this typo be put on this rectors dictonary?

Reply:
Hi, I'm not aware of such rule. Probably tutorial only